.travis.yml

```diff
@@ -40,13 +40,3 @@ jobs:
   - os: osx
     language: generic
     env: PYTHON=3.6.6
-  - stage: release
```

Comment:
https://github.com/amzn/MXFusion/blob/97d3547dc0f40072189bf2d0d529e7a4c7055d7a/.travis.yml#L44-L53

Please change tab to space